svelte/components/Global/Home.svelte

```diff
@@ -63,9 +70,18 @@
   <span slot="content">
     {#if chainId && account && address}
       {#if tokenID}
-        <Nft {chainId} {address} {tokenID} />
+        <Nft {chainId} {address} bind:tokenID {account} {platform} />
```

Comment:
tokenID not changed inside Nft component, use {tokenID}, no bind:

variable bind twice is very dangerous, can easily produce dead locks

Reply:
have to see if we can manage nft display in another way

Reply:
now it use svelte store currentTokenID